skunkbooth/filters/grayscale.py

```diff
@@ -0,0 +1,29 @@
+from PIL import Image
+from PIL.ImageOps import grayscale
+
+
+class filter:
+    """Sample filter for PIL operations"""
```

Comment:
The file you based this on was a sample filter, but for this filter you've written this should be changed to describe the filter.

Reply:
That does make more sense, haha.